lib/components/SuperCombo.js

```diff
@@ -314,6 +315,16 @@ function MLSuperCombo$update() {
     this._comboAfter.el.style.height = afterHeight > 0 ? afterHeight + 'px' : '0px';
 }
 
+/**
+ * Component instance method
+ * Sets the selected value and data to provided value
+ * @param {Variable} obj
```

Comment:
Oops, sorry I didn't catch this before - Is this documentation wrong (or am I just misreading?)...

Should it be:

`@param {Object} value`

Reply:
Fixed in https://github.com/milojs/milo-ui/pull/90/commits/8fc3bd4aed37e635988d797ad2729436d63a9524#diff-fc029ec02677b30bde05b6df429366f6L389, however I foolishly `--amend`ed it because I forgot about the dist commit.